.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.4"
+  - "3.5"
+install: pip install coveralls
+ # - if [ "$TRAVIS_PYTHON_VERSION" == "3.2" ]; then travis_retry pip install coverage==3.7.1; fi
+ # - if [ "$TRAVIS_PYTHON_VERSION" != "3.2" ]; then travis_retry pip install coverage; fi
+script: coverage run neubot_runtime/test_utils_path.py 
```

Comment:
could you find a way to teach coverage to run all the test scripts present below the neubot_runtime directory?